lib/vector-core/src/event/value.rs

```diff
@@ -83,10 +83,6 @@ impl TryFrom<TomlValue> for Value {
     }
 }
 
-// We only enable this in testing for convenience, since `"foo"` is a `&str`.
-// In normal operation, it's better to let the caller decide where to clone and when, rather than
-// hiding this from them.
-#[cfg(test)]
 impl From<&str> for Value {
```

Comment:
Do we have to drop the `#[cfg(test)]` to make this build?

Reply:
We do. Someone more clever with cargo than me might be able to get the `test` to propagate down into this package, but I wasn't able to figure out how to do it. Ends up being kind of a bummer to have this present in non-test.